### DISCUSSION TITLE
Question about concepts

### DISCUSSION THREAD

**Discussion by specblades** [TheLastBen/fast-stable-diffusion#962] [Q&A]
How should I name the files to separate the concepts?

**Comment by TheLastBen** [comment]
Just upload the pictures, as long as the resolution is 512 or above, no need to rename them

**Reply by specblades** [reply]
So, if i want to train on four different concepts, should i upload only four images?

**Reply by TheLastBen** [reply]
concept images help reduce overfitting, keep the number of concept images above 50

**Reply by specblades** [reply]
Okay, that makes a little more sense. Say, suppose my training dataset has both hypothetical cats and dogs. Should I divide the concept dataset exactly in half for cats and dogs?

**Reply by TheLastBen** [reply]
no need, just upload as much images as you want to help widen the concept of dogs and cats in case the model is overfit

**Reply by SAC020** [reply]
Do concept images need to be square? 

Is concept images captioning relevant?

**Reply by TheLastBen** [reply]
No need to resize or caption them.

**Reply by SAC020** [reply]
> No need to resize

Got that, but do they need to be square? Reason for asking is, if I'm using non-square subject images, then the gens get deformed - does the same apply to concept images?

**Reply by TheLastBen** [reply]
it will autocrop them

**Reply by gilroff** [reply]
> it will autocrop them

If one or two concept images are below 512px, will it be a problem?

**Reply by TheLastBen** [reply]
not a problem, you can work with even smaller pics

**Reply by gilroff** [reply]
Ok thank you <3
